extract.sh

```diff
@@ -23,11 +23,11 @@ if [ ! -e ${data_root}/rsd ]; then
     echo "missing ${data_root}/rsd"
     exit 1
 fi
-# dir_count=$(ls ${data_root} | wc -l)
-# if [ "$dir_count" -ne "2" ]; then
-#     echo "unexpected dirs under ${data_root}; only ltf nd rsd allowed"
-#     exit 1
-# fi
+dir_count=$(ls ${data_root} | wc -l)
+if [ "$dir_count" -ne "2" ]; then
+    echo "unexpected dirs under ${data_root}; only ltf nd rsd allowed"
```

Comment:
@limanling please keep this check.  it would have helped us to figure out that we had different input.

Reply:
I see. Thank you!